### PR TITLE
Added a print for hello zainab

### PR DESCRIPTION
**Pull request by wanda-carlson**
Hello, this is the PR body

### CODE REVIEW

**Comment by henry-richard** [comment]
This is cool!

**Comment by bert-griffin** [comment]
Does it sync both ways?

File: src/bert.py [hunk 1]
```diff
@@ -10,3 +10,4 @@
 print("Hello Zak!")
 print("Hello Zainab!")
 print("Hello again, Zainab!")
```
Comment (henry-richard) [inline src/bert.py:12]: comment 1

File: src/bert.py [hunk 2]
```diff
@@ -10,3 +10,4 @@
...
+print("Hello again, Zainab!")
```
Comment (henry-richard) [inline src/bert.py:13]: comment 2

**Review by henry-richard — CHANGES_REQUESTED** [review]
leaving a comment

**Review by henry-richard — APPROVED** [review]

**Pull request merged**